id.go

```diff
@@ -8,12 +8,12 @@ import (
 	"strings"
 )
 
-func _IdFromString(s string) (shard int, realm int, num int, checksum *string, err error) {
+func _IdFromString(s string) (shard int, realm int, num int, checksum *string, alias *PublicKey, err error) {
```

Comment:
We really should not support `PublicKey` for all `ID` types

Reply:
I need it here for AccountID, others ignore the alias